### PR TITLE
[docs] update contributing guide

### PR DESCRIPTION
**Pull request by jamiedemaria**
### Summary & Motivation
guide for contributing docs was out of date and missing some new info. this updates it

### CODE REVIEW

**Review by yuhan — APPROVED** [review]

File: docs/content/community/contributing.mdx [hunk 1]
```diff
@@ -147,19 +147,28 @@ To run the Dagster documentation website locally, run the following commands:
...
-make build
+make apidoc-build
 ```
 
+The majority of our code snippets are pulled from real python files. This allows us to test our code snippets and ensure they remain up-to-date. In `.mdx` files you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. You can find the corresponding python file at `dagster/examples/docs_snippets/docs/snippets/path/to/file.py`. The code included in each snippet is the code in the file between the `# start_marker` and `# end_marker` comments. If you change a code snippet, run:
+
+```bash
+make mdx-format
+```
+from the `docs` directory.
```
Comment (erinkcochran87) [inline docs/content/community/contributing.mdx:168]:
```suggestion
The majority of our code snippets are pulled from real Python files. This allows us to test our code snippets and ensure they remain up-to-date. 

In `.mdx` files, you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. For example:

[add example here]

You can find the corresponding Python file at `dagster/examples/docs_snippets/docs/snippets/path/to/file.py`. The code included in each snippet is the code in the file between the `# start_marker` and `# end_marker` comments. 

If you change a code snippet, run the following from the `docs` directory:

```bash
make mdx-format
```
```

Capitalization and some formatting. The paragraph was a bit dense so I broke it up to make it easier to skim.

Consider adding an example of the code snippets here to demonstrate what's being discussed, too.

**Comment by jamiedemaria** [comment]
@erinkcochran87 i'm not sure how to add an example of what the code block should look like without causing the docs snippets tests to fail. if i add an example like this

```
python file=/path/to/file.py startafter=start endbefore=end

def some_code():
pass
```

with the intent being that the rendered code block on the page actually shows the `python file...` bit, the docs snippets tests fail because it thinks there should be a file the snippet is referencing. i'm not really sure what to do about that

File: docs/content/community/contributing.mdx [hunk 2]
```diff
@@ -160,12 +160,18 @@ If you change any `.rst` files, be sure to run the following command in the `doc
 make apidoc-build
 ```
 
-The majority of our code snippets are pulled from real python files. This allows us to test our code snippets and ensure they remain up-to-date. In `.mdx` files you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. You can find the corresponding python file at `dagster/examples/docs_snippets/docs/snippets/path/to/file.py`. The code included in each snippet is the code in the file between the `# start_marker` and `# end_marker` comments. If you change a code snippet, run:
+The majority of our code snippets are pulled from real Python files. This allows us to test our code snippets and ensure they remain up-to-date. 
+
+In `.mdx` files, you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. For example:
+
+[add example here]
```
Comment (erinkcochran87) [inline docs/content/community/contributing.mdx:167]: Example?

Comment (jamiedemaria) [reply docs/content/community/contributing.mdx:167]: i'm not sure how to get the `python file.....` bit to actually show in the code block, the docs snippet tests complain about it

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**